beacon-chain/core/validators/validator.go

```diff
@@ -9,10 +14,13 @@ import (
 	"github.com/prysmaticlabs/prysm/shared/bitutil"
 	"github.com/prysmaticlabs/prysm/shared/hashutil"
 	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/sirupsen/logrus"
 )
 
 const bitsInByte = 8
 
+var log = logrus.WithField("prefix", "core/validators")
```

Comment:
no need to add log here (we are keeping these packages stateless and no assumptions about where they'll be run. we can keep logs in services)

Reply:
that makes sense